graylog2-server/src/main/java/org/graylog2/plugin/BaseConfiguration.java

```diff
@@ -65,6 +66,9 @@
     @Parameter("message_journal_enabled")
     private boolean messageJournalEnabled = true;
 
+    @Parameter(value = "message_journal_mode", validators = StringNotBlankValidator.class)
```

Comment:
this could be empty if the journal is disabled

Reply:
You mean someone would specify it like this?
```
message_journal_enabled = false
message_journal_mode =
```
That will fail indeed. Just omitting the option entirely will work:
```
message_journal_enabled = false
```
I'll make it more resilient then to support the first case.